src/main/archetype/ui.frontend/src/main/webpack/components/content/navigation/scss/navigation.scss

```diff
@@ -0,0 +1,28 @@
+/* BEM Description
+BLOCK cmp-navigation
+    ELEMENT cmp-navigation__group
+    ELEMENT cmp-navigation__item
+        MOD cmp-navigation__item--active
+        MOD cmp-navigation__item--level-*
+    ELEMENT cmp-navigation__item-link
+*/
+
+.cmp-navigation {
+  &__group {
+
+  }
+
+  &__item {
+    &--active {
+
+    }
+
+    //&--level-1 {
```

Comment:
would probably provide `.cmp-navigation__item--level-0` as the example just to make it clear that it's zero indexed.